Tests/SonarScanner.MSBuild.Tasks.UnitTest/WriteProjectInfoFileTests.cs

```diff
@@ -578,6 +570,29 @@ public void GetProjectGuid_WhenSolutionConfigurationContentsHasMultipleMatch_Ret
             actual.Should().Be(expectedGuid);
         }
 
+        [TestMethod]
+        public void GetProjectGuid_InvalidPath_DoesNotThrow()
+        {
+            var expectedGuid = "{10F2915F-4AB3-4269-BC2B-4F72C6DE87C8}";
+            var notValidPath = @"D:\a\1\s\src\https://someUrl.me:1623";
+            var fullProjectPath = @"C:\NetStdApp\NetStdApp.csproj";
+            var testSubject = new WriteProjectInfoFile
+            {
+                ProjectGuid = null,
+                FullProjectPath = fullProjectPath,
+                SolutionConfigurationContents = @"<SolutionConfiguration>
```

Comment:
do we have a test also when there are two entries for the same path (at least to document the behavior)?